web/src/HeroScreen.stories.tsx

```diff
@@ -1,8 +1,17 @@
 import React from "react"
 import HeroScreen from "./HeroScreen"
+import SocketBar from "./SocketBar"
+import { SocketState } from "./types"
 
 export default {
-  title: "New UI/_To Review/HeroScreen",
+  title: "HeroScreen",
```

Comment:
i think we want to keep this under the `New UI` category, otherwise it's grouped outside the rest of the stories.

Reply:
sgtm. tho i feel like we need to redo the organization here - it's kind of a mess. always think it's sus to call something "New" because that will always be inaccurate someday.

Reply:
yeah, i won't argue with you there! +1 on reorganizing (and refreshing) the storybook entries.